Osiris/Hacks/Misc.cpp

```diff
@@ -412,6 +426,24 @@ void Misc::drawBombTimer() noexcept
     }
 }
 
+void Misc::drawFov() noexcept
+{
+    auto local = localPlayer.get();
```

Comment:
You dont need that line of code, localPlayer, like below.

    if (!localPlayer->isAlive()) return;

Reply:
its ok